logcheck/README.md

```diff
@@ -85,6 +85,11 @@ runtime in klog.
 
 This check flags all invocation of `klog.V(0)` or any of it's equivalent as errors
 
+## key (disabled by default)
```

Comment:
We should make it "enabled by default". I missed that when looking at the code - documentation for the win!

Reply:
I remember last time the discussion was  disable  by default.
Are we sure it's enabled by default now?

Reply:
That was when I thought that the new check would be just for special characters, i.e. a new requirement.

But now it also covers "must be string constant", which was logcheck checking for by default before. We should not make logcheck check less than before when updating it.